tests/chainer_tests/function_hooks_tests/test_cupy_memory_profile.py

```diff
@@ -153,6 +153,16 @@ def test_reentrant_total_bytes(self):
         self.assertEqual(self.h.total_acquired_bytes(), 1024)
 
 
+@testing.parameterize(
+    {'unit': 'B'},
+    {'unit': 'KB'},
+    {'unit': 'MB'},
+    {'unit': 'GB'},
+    {'unit': 'TB'},
+    {'unit': 'PB'},
+    {'unit': 'EB'},
+    {'unit': 'ZB'},
```

Comment:
Could you add testcases for `'auto'` and `'auto_foreach'`?

Reply:
Thank you! I had forgotten and added those cases.